src/main/java/cloud/fogbow/rcs/core/intercomponent/RemoteFacade.java

```diff
@@ -25,6 +35,29 @@ public String requestService(String senderId, ServiceType serviceType) {
         return this.factory.makeCatalogService().requestService(senderId, serviceType);
     }
 
+    public List<ServiceType> getServices() {
+        List<String> possibleServicesProperties = new ArrayList<String>(){
+            {
+                add(FNS_SERVICE_PROPERTY);
+                add(RAS_SERVICE_PROPERTY);
+                add(AS_SERVICE_PROPERTY);
+                add(MS_SERVICE_PROPERTY);
+            }
+        };
+
+        List<ServiceType> services = new ArrayList<>();
+
+        for(String serviceProperty : possibleServicesProperties) {
```

Comment:
not sure if we want to support something different, but i think with this implementation we're limiting the services definition to one type per site, right?

Reply:
nope, we're getting all services types from the site...

Reply:
e.g if the site has AS, RAS and MS, they 3 are going to be returned from the xmpp call

Reply:
i see. what if the site has two different AS instances and both are supposed to be on the catalog. do we want to support that?